src/Umbraco.Infrastructure/Persistence/Repositories/Implement/CreatedPackageSchemaRepository.cs

```diff
@@ -736,8 +736,8 @@ void OnSerializedMedia(IMedia media, XElement xmlMedia)
             // get the media file path and store that separately in the XML.
             // the media file path is different from the URL and is specifically
             // extracted using the property editor for this media file and the current media file system.
-            Stream mediaStream = _mediaFileManager.GetFile(media, out var mediaFilePath);
-            if (mediaFilePath is not null)
+            Stream? mediaStream = _mediaFileManager.GetFile(media, out var mediaFilePath);
+            if (mediaStream != null && mediaFilePath is not null)
```

Comment:
The private `PackageMedia()` method in `PackagesRepository` and `CreatedPackageSchemaRepository` does the same thing, but doesn't handle nullable in same way.

Since the method in `PackagesRepository` is obsolete I copied the logic from the new method in `CreatedPackageSchemaRepository`.

Reply:
`_mediaFileManager.GetFile()` will _never_ be null, it cannot be null (if it _could_ we'd get a compiler warning here!)

The null check is unnecessary.